_includes/live-demos/drive-standalone-pick-controls/index.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+  var button = document.querySelector('.tinydrive-standalone-demo-pick');
+  var input = document.querySelector('.tinydrive-standalone-demo-input');
+
+  button.addEventListener('click', function () {
+    var controls = tinydrive.pick({
+      demo_files_url: '{{ site.baseurl }}/demo/tiny-drive-demo/demo_files.json',
+      token_provider: function (success) { success({ token: 'fake-token' }); },
+    }, function(result) {
+      /* input.value = result[0].url; */
+      input.value = 'https://drive.tiny.cloud/1/h0ed4hzbe69rfcrnj0uwvwi73e8y5m70jlolaalzkssog63b/48bda29d-ed1f-488f-adf7-b597dd3a8791';
```

Comment:
Why are we hardcoding this like that?

Reply:
I have no idea. the `drive-standalone-pick` did it like it, and I did not feel comfortable enough in the reason to change it.

Reply:
And i don't know why either. It's in the original example and will be hidden behind the `example.js`